examples/coap-client/laze.yml

```diff
@@ -2,7 +2,7 @@ apps:
   - name: example-coap-client
     env:
       global:
-        CARGO_ENV:
-          - CONFIG_ISR_STACKSIZE=32768
+        executor_stacksize_required:
+          - "32768"
```

Comment:
Having to quote here is unfortunate. IIUC this is due to an architecture quirk in serde: https://github.com/dtolnay/serde-yaml/issues/388

Reply:
I'd have been surprised if the number worked in the first place, considering that environment variables are generally stringly typed.